src/remote-config/remote-config-api-client.ts

```diff
@@ -237,6 +237,9 @@ export class RemoteConfigApiClient {
   public listVersions(options?: ListVersionsOptions): Promise<ListVersionsResult> {
     if (typeof options !== 'undefined') {
       options = this.validateListVersionsOptions(options);
+      // Remove undefined fields from options
+      Object.keys(options).forEach(key =>
+        (options as any)[key] === undefined && delete (options as any)[key])
```

Comment:
I feel like this logic belongs in the `validateListVersionsOptions` method.

Also `typeof ... === 'undefined'`

Reply:
Good call. Moved to `validateListVersionsOptions`